demisto_sdk/__main__.py

```diff
@@ -479,6 +484,7 @@ def validate(config, **kwargs):
             debug_git=kwargs.get('debug_git'),
             include_untracked=kwargs.get('include_untracked'),
             quite_bc=kwargs.get('quite_bc_validation'),
+            run_with_multiprocessing=run_with_mp,
```

Comment:
`multiprocessing` or reversing into `single_process` would be better/shorter names

Reply:
done